docs/resources/swr_image_trigger.md

```diff
@@ -0,0 +1,115 @@
+---
+subcategory: "Software Repository for Container (SWR)"
+---
+
+# huaweicloud_swr_image_trigger
+
+Manages a SWR image trigger within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "organization_name" {}
+variable "repository_name" {}
+variable "cluster_id" {}
+variable "namespace" {}
+variable "name" {}
+
+resource "huaweicloud_swr_image_trigger" "test"{
+  organization    = var.organization_name
+  repository      = var.repository_name
+  workload_type   = "deployments"
+  workload_name   = "test_name"
+  cluster_id      = var.cluster_id
+  namespace       = var.namespace
+  condition_value = ".*"
+  enabled         = "true"
+  name            = var.name
+  condition_type  = "all"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this parameter will create a new resource.
+
+* `organization` - (Required, String, ForceNew) Specifies the name of the organization.
+
+  Changing this parameter will create a new resource.
+
+* `repository` - (Required, String, ForceNew) Specifies the name of the repository.
+
+  Changing this parameter will create a new resource.
+
+* `workload_type` - (Required, String, ForceNew) Specifies the type of the application.
+  Value options: **deployments**, **statefulsets**.
+
+  Changing this parameter will create a new resource.
+
+* `workload_name` - (Required, String, ForceNew) Specifies the name of the application.
+
+  Changing this parameter will create a new resource.
+
+* `namespace` - (Required, String, ForceNew) Specifies the namespace where the application is located.
+
+  Changing this parameter will create a new resource.
+
+* `condition_value` - (Required, String, ForceNew) Specifies the trigger condition value. Value options:
+  + When condition_type is set to `all`, set this parameter to `.*`.
+  + When condition_type is set to `tag`, set this parameter to specific image tags separated by semicolons (;).
+  + When condition_type is set to `regular`, set this parameter to a regular expression.
+
+  Changing this parameter will create a new resource.
+
+* `enabled` - (Required, String) Specifies whether to enable the trigger. Value options **true**, **false**.
+
+* `name` - (Required, String, ForceNew) Specifies the trigger name.
+
+  Changing this parameter will create a new resource.
+
+* `condition_type` - (Required, String, ForceNew) Specifies the trigger condition type.
```

Comment:
it's better to put `condition_type` and `condition_value` together.

Reply:
done